docs/data_management/transfer/tutorial/globus_group_management.md

```diff
@@ -0,0 +1,273 @@
+
+1. If you clicked the "Accept Invitation" button, you will be taken to a form. Fill out the form and click "Accept Invitation" to join the group. Other groups may have additional fields to fill out.
+
+    ![Accept invitation form.](./images/gg-accept-group/003-form.png)
+
+## How Do I Leave a Group?
+
+1. [Get to the Groups page of the Globus Web App](#where-can-i-find-the-groups-page-in-the-globus-web-app).
+1. [Find the Group you wish to leave](./globus_group_management.md#how-do-i-find-a-group). It may help to check the "MY GROUPS" Checkbox to only show Groups you are a member of.
+
+    ![My Groups checkbox checked](./images/common/groups-checked.png)
+
+1. Click the "Leave Group" button. Note that this action cannot be undone, but you may [Rejoin the Group](#how-do-i-request-to-join-a-group) or be [Invited to Rejoin](#how-do-i-find-and-accept-an-invitation-from-a-group).
+
+    ![Leave group button on the groups page.](./images/gg-leave-group/001-leave-button.png)
+
+## How Do I Create a Group?
+
+1. [Get to the Groups page of the Globus Web App](#where-can-i-find-the-groups-page-in-the-globus-web-app).
+1. Click the "Create New Group" button at the top-right of the window.
+
+    ![Create new Group button on Groups page](./images/gg-create-group/001-create-group-button.png)
+
+1. Fill in the form. More information about each field is below. When you have finished, click the "Create Group" button to create the group.
+
+    If you need to change the Group settings later, you can [Modify the Group](#how-do-i-modify-a-group-or-subgroup).
+
+    1. **Group Name**: Enter a group name here.
+    1. **Description** (optional): Describe the purpose and scope of the group here.
+    1. **Terms & Conditions** (optional): should not be necessary for UAB groups.
+    1. **Enable High Assurance** checkbox: Check if your group will deal with PHI, HIPAA, or other protected data.
+    1. **Users may request to join this group** checkbox: Uncheck to make the group invite-only. Core administrators should consider leaving this box checked for their top-level group to avoid work filtering invitations and reduce noise.
+    1. **Signup Fields** buttons: First Name, Last Name, and Organization are always required. We recommend also selecting "Project", "Department", and "Field of Science". Feel free to select any fields you wish.
+    1. **Group Visibility** radio buttons: Click "Members of this Group" to hide the group from non-members.
+    1. **Membership Visibility** radio buttons: Click "Administrators and Managers Only" to hide member names from each other. Core adminstrators should consider using the "Administrators and Managers Only" option for their top-level groups because it improves the user experience when members are looking for administrator names.
+
+    ![Create Group form.](./images/gg-create-group/002-create-group-form.png)
```

Comment:
The image is labeled with numbers, but the text descriptions above uses letters. Is it possible to use either numbers or letters consistently?

Reply:
This is a consequence of having an ordered list nested in an ordered list. I'll fix it by using an unordered list with numbers written out.